### DISCUSSION TITLE
DataSync

### DISCUSSION THREAD

**Discussion by diofathur** [awsugid/forum#25] [General]
Halo teman2 semuanya,

saya mau tanya jadi kita ada case mau menggunakan datasync jadi datasync ini kami gunakan untuk melakukan sync antara virtual machine storage yang berada di environment GCP kami dan ingin di sync ke amazon s3 
berikut gambar HLD nya saya sematkan dibawah
![WhatsApp Image 2023-11-06 at 11 23 07_e07707a7](https://github.com/awsugid/forum/assets/84631782/6edfdd22-34a7-4031-a323-f1fb1871f4d9)

soalnya saya baca2 di docs nya aws itu cma ada contoh dari cloud storage gcp ke amazon s3
https://docs.aws.amazon.com/datasync/latest/userguide/tutorial_transfer-google-cloud-storage.html

dan ada tapi dari on prem ke amazon s3 
https://docs.aws.amazon.com/datasync/latest/userguide/s3-cross-account-transfer.html

mungkin ada yang pernah kah menemukan case yang seperti teman2 ? 

Terima kasih

**Comment by rigelroubbahari** [comment]


**Reply by diofathur** [reply]
Haloo kalo untuk pemula untuk tech nya mungkin bisa belajar vpc ec2 s3 iam trus rds dan security2nya untuk nonnya cost mngkin biar aware terhadap cost optimization soalnya itu termasuk penting jga,

Mngkin bsa ikut yang cloud practitioner di udemy

**Reply by rigelroubbahari** [reply]
Terimaksih mas infonya🙏

**Reply by diofathur** [reply]
Yoo sama2, sama mngkin bsa belajar contoh2 case sih trus implementin di aws
